src/com/twitter/intellij/pants/settings/PantsSettings.java

```diff
@@ -76,6 +76,10 @@ public boolean isEnableIncrementalImport() {
     return getLinkedProjectsSettings().stream().anyMatch(PantsProjectSettings::isEnableIncrementalImport);
   }
 
+  public boolean isUseIntellijCompiler() {
+    return getLinkedProjectsSettings().stream().anyMatch(s -> s.useIntellijCompiler);
+  }
```

Comment:
Does this mean that if any of my imported projects use the intellij compiler, all of them have to? I'm not sure what getLinkedProjectSettings does.

Reply:
One IntelliJ window can contain multiple projects actually. So yes, that's what this means. 

This mostly just follows the pattern there. and the tricky part is that if a source file can belong to multiple projects, then which compiler should we use? I think this should be an okay heuristics.